src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -43,10 +43,6 @@
     <DotNetCliToolReference Include="dotnet-sourcelink-git" Version="2.1.2" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
```

Comment:
Our test.csproj doesn't test transactions with framework net461. Could you test that scenario after removing this ItemGroup?

Reply:
You could also just remove 
```<compile Remove="..\..\test\Microsoft.Azure.ServiceBus.UnitTests\TransactionTests.cs" />```
and let appveyor test it out for you.

Reply:
> let appveyor test it out for you.

I can't. PRs seem not to be configured to run the tests when those are coming from external collaborators. And I can't see the project on AppVeyor to trigger the build. Want to fix that @nemakam 😉

Reply:
@SeanFeldman 
Not sure I get you?
When you send a PR, app veyor automatically runs all the tests. You may not be able to trigger a new one manually, but you could always see the build. Anything I'm missing?

Reply:
Is this just for this PR or have you never seen the appveyor link for any PR? 
Because every PR does trigger appveyor.
https://ci.appveyor.com/project/vinaysurya/azure-service-bus-dotnet/build/3.0.2 <-- This is the link to the last run which had 2 failed tests